client/checkout/classic/upe.js

```diff
@@ -7,12 +7,16 @@ import './style.scss';
 import {
 	PAYMENT_METHOD_NAME_CARD,
 	PAYMENT_METHOD_NAME_UPE,
+	COOKIE_CART_HASH,
+	COOKIE_UPE_PAYMENT_INTENT,
+	COOKIE_UPE_SETUP_INTENT,
 } from '../constants.js';
 import { getConfig, getCustomGatewayTitle } from 'utils/checkout';
 import WCPayAPI from '../api';
 import enqueueFraudScripts from 'fraud-scripts';
 import { getFontRulesFromPage, getAppearance } from '../upe-styles';
 import { getTerms } from '../utils/upe';
+import { getCookieValue } from '../../utils';
```

Comment:
Pedantic peeve, but you could have put this function inside `../utils/upe.js`.

Reply:
Hehe, yes!

Reply:
On another thought, `getCookieValue` is a generic utility not limited to UPE. Perhaps code outside of UPE might benefit from this generic function at some point?

Reply:
On its own `getCookieValue` is a pretty generic looking utility, although, that being said, I don't think it has any purpose outside of the UPE. At least, I don't think there is a reason to maintain a static session for payment intents without the UPE, since the intent will regularly never exist until the `Pay for order` button is selected.

I did originally admit that this was a pedantic complaint and honestly the only reason I made it was because it took me an extra five seconds to locate this function. 😅 I'm not too bothered where it ends up, so I'll leave its final resting place up to your own discretion.

Reply:
Thanks :) I'll leave it in the common utils for now so if anyone is looking to access any cookie they have an easier way than parsing `document.cookie`.